metadata.json

```diff
@@ -115,5 +115,5 @@
   "description": "Standard Library for Puppet Modules",
   "pdk-version": "2.3.0",
```

Comment:
isn't pdk-version supposed to be 2.4.0?

Reply:
ah, the version updated in between the original commit and when I added the fixes and exclusions. Since this one required no exclusions a second pdksync wasn't run